lib/pagseguro/refund.rb

```diff
@@ -12,8 +12,8 @@ class Refund
     attr_accessor :value
 
     # Calls the PagSeguro webservice and register the refund.
+    # Return boolean.
     def register
-      params = Serializer.new(self).to_params
       Response.new(Request.post("transactions/refunds", api_version, params))
```

Comment:
@lucasrenan Are you sure? Current code (aka, before this PR) says to be a `PagSeguro::Refund::Response` ( https://github.com/pagseguro/ruby/blob/2.4.0-dev/spec/pagseguro/refund_spec.rb#L38-L44 ), and the previous specs the same thing.